src/components/ListAllArticles.jsx

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios';
+import ArticleCard from './ArticleCard';
+import { useEffect, useState } from 'react';
+import getArticles from '../utils/getArticles';
+const URLpath = "https://ng-news.onrender.com/api/articles"
+
+const ListAllArticles = () => {
+    const [articles, setArticles] = useState([])
+    const [isLoading, setIsLoading] = useState(true)
+
+    useEffect(() => {
+        getArticles()
+        .then((articles) => {
```

Comment:
we also want to setloading to false in the catch block so that our screen isnt permanently loading if it runs into an error, as this tells the user they need to keep waiting